doc/radosgw/cloud-transition.rst

```diff
@@ -288,50 +299,63 @@ For example,
             }
         }
     ]
+```
 
-Object modification & Limitations
+Object Modification and Limitations
 ----------------------------------
 
-The cloud storage class once configured can then be used like any other storage class in the bucket lifecycle rules. For example,
-
-::
+The cloud storage class once configured can then be used like any other storage
+class in bucket lifecycle (LC) rules. For example,
 
+```
     <Transition>
       <StorageClass>CLOUDTIER</StorageClass>
       ....
       ....
     </Transition>
+```
 
+Since the transition is unidirectional, when configuring S3
+lifecycle rules, the cloud storage class should be specified
+last among all the storage classes the object transitions to.
+Subsequent rules (if any) do not apply post-transition to the cloud.
 
-Since the transition is unidirectional, while configuring S3 lifecycle rules, the cloud storage class should be specified last among all the storage classes the object transitions to. Subsequent rules (if any) do not apply post transition to the cloud.
-
-Due to API limitations there is no way to preserve original object modification time and ETag but they get stored as metadata attributes on the destination objects, as shown below:
-
-::
+Due to API limitations there is no way to preserve the original object
```

Comment:
s/Due to API limitations/Due to API limitations,/

(Add a comma after this introductory phrase.)